### ISSUE TITLE
fix: bad state: The 'read' method can only be called once

### ISSUE THREAD

**Issue by felangel** [VeryGoodOpenSource/dart_frog#27]
**Description**

Seeing the following:

```sh
Asynchronous error
Bad state: The 'read' method can only be called once on a shelf.Request/shelf.Response object.

package:shelf/shelf_io.dart 143:11  handleRequest
```

When running the example and visiting `http://localhost:8080` in the browser.

**Comment by felangel** [comment]
Looks like it was a false positive due to a rogue service worker. Closing for now.

**Comment by MisterJimson** [comment]
I'm able to repro this consistently by hitting 404 routes.

Steps
```
dart_frog create todo
cd todo
dart_frog dev
Visit http://localhost:8080/abc
Visit http://localhost:8080/abc again
```

**Comment by felangel** [comment]
@MisterJimson thanks for reporting! I’ll take a closer look shortly 👍

**Comment by MisterJimson** [comment]
It's reproducible with Postman GET requests, I'll try curl in a moment.

**Comment by MisterJimson** [comment]
cURL as well

**Comment by felangel** [comment]
@MisterJimson should be resolved in dart_frog v0.0.1-dev.4 🎉
